docs/public/installation/zonemaster-gui.md

```diff
@@ -266,14 +266,14 @@ Make sure Zonemaster-GUI is properly installed.
    its page footer.
 
 3. Verify that when you mouse over this text the versions of the following
-   Zonemaster components are shown: Backend, Engine and GUI.
+   Zonemaster components are shown: Backend, Engine, LDNS and GUI.
```

Comment:
It is not the version of LDNS, it is Zonemaster-LDNS. LDNS is another software. I suggest:
```
components are shown: Zonemaster-LDNS, Zonemaster-Engine, Zonemaster-Backend and Zonemaster-GUI.
```

Reply:
Done.